app/src/androidTest/java/org/mozilla/focus/helpers/TestHelper.kt

```diff
@@ -39,6 +39,7 @@ object TestHelper {
     @JvmField
     var mDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
     const val waitingTime = DateUtils.SECOND_IN_MILLIS * 15
+    const val waitingTimeShort = DateUtils.SECOND_IN_MILLIS * 5
```

Comment:
I'm OK with adding this back if we think it won't cause any Firebase issues with short timeouts. I figure if we wait longer on Firebase there's less chance of timing problems, that was my thinking when I removed it in the refactor - I was wondering the benefit. I'll leave it up to you.

Reply:
I thought we can use this when we don't expect a long wait as for page loading, if some view doesn't show in 5 sec, maybe it's not worth waiting longer thus increasing the total time of the test run. But I don't mind following your advice, as you think is better.